samples/user-interface/appwidgets/src/main/java/com/example/platform/ui/appwidgets/glance/weather/WeatherGlanceWidget.kt

```diff
@@ -81,8 +82,12 @@ class WeatherGlanceWidget : GlanceAppWidget() {
         setOf(thinMode, smallMode, mediumMode, largeMode)
     )
 
+    override suspend fun provideGlance(context: Context, id: GlanceId) {
+        provideContent { Content() }
+    }
+
     @Composable
-    override fun Content() {
+    fun Content() {
         // Get the stored stated based on our custom state definition.
         val weatherInfo = currentState<WeatherInfo>()
```

Comment:
We probably should simplify the state management now that recomposition is supported but can be done in a follow up PR